### PR TITLE
[new] Added deferer@1.0.0

### PR DESCRIPTION
**Pull request by dominic**

### CODE REVIEW

File: ajax/libs/deferer/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "repository": "https://github.com/dominic/deferer",
+  "main": "deferer.js",
+  "license": "MIT",
+  "dependencies": {},
+  "npmName": "deferer",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "*/"
```
Comment (PeterDaveHello) [inline ajax/libs/deferer/package.json]: Should not include everything here.

Comment (dominic) [inline ajax/libs/deferer/package.json]: Fair; that's the output of `cdnjs-importer`, but the example in the README is different. Sorry about that.

You can just close this if it's waiting on notoriety though. No point keeping a PR around for that.

Comment (PeterDaveHello) [inline ajax/libs/deferer/package.json]: @dominic, no need to sorry.
And it's okay to open this until maybe you got a number > 50 of watch + star + fork

cc @IonicaBizau

Comment (IonicaBizau) [inline ajax/libs/deferer/package.json]: @dominic @PeterDaveHello The cdnjs-importer searches for a directory like `src`, `dist` or `lib` -- if nothing is found, then it fallsback to the root directory. Any idea how can we improve that?

Comment (PeterDaveHello) [inline ajax/libs/deferer/package.json]: Yep: main: `deferer.js` -> grab `deferer*.js`, more safer!

**Comment by PeterDaveHello** [comment]
Hello, any updates here? Thanks.

**Comment by dominic** [comment]
Nah, happy to close.

**Pull request closed without merging**